apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue

```diff
@@ -214,33 +189,33 @@ const handleClickSignOut = async () => {
-                             class="currency-menu-wrapper"
-                        >
-                            <div class="sub-menu-wrapper">
-                                <div v-for="(item, index) in state.currencyMenuItems"
-                                     :key="index"
-                                     class="sub-menu"
-                                     @click.stop="handleCurrencyClick(item.name)"
-                                >
-                                    <span>{{ item.label }}</span>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                    <p-i :name="state.currencyMenuVisible ? 'ic_chevron-up' : 'ic_chevron-down'"
-                         class="arrow-icon"
-                         width="1rem"
-                         height="1rem"
-                    />
-                </div>
+                <!--                <div v-on-click-outside="handleClickOutsideCurrencyMenu"-->
+                <!--                     class="info-menu currency"-->
+                <!--                     @click.stop="handleCurrencyDropdownClick"-->
+                <!--                >-->
+                <!--                    <span class="label">{{ $t('COMMON.GNB.ACCOUNT.LABEL_CURRENCY') }}</span>-->
+                <!--                    <div class="value">-->
+                <!--                        <span>{{ state.currency }}</span>-->
+                <!--                        <div v-if="state.currencyMenuVisible"-->
+                <!--                             class="currency-menu-wrapper"-->
+                <!--                        >-->
+                <!--                            <div class="sub-menu-wrapper">-->
+                <!--                                <div v-for="(item, index) in state.currencyMenuItems"-->
+                <!--                                     :key="index"-->
+                <!--                                     class="sub-menu"-->
+                <!--                                     @click.stop="handleCurrencyClick(item.name)"-->
+                <!--                                >-->
+                <!--                                    <span>{{ item.label }}</span>-->
+                <!--                                </div>-->
+                <!--                            </div>-->
+                <!--                        </div>-->
+                <!--                    </div>-->
+                <!--                    <p-i :name="state.currencyMenuVisible ? 'ic_chevron-up' : 'ic_chevron-down'"-->
+                <!--                         class="arrow-icon"-->
+                <!--                         width="1rem"-->
+                <!--                         height="1rem"-->
+                <!--                    />-->
+                <!--                </div>-->
```

Comment:
question: r u going to remove this later?

Reply:
Nope, This code might need recovery in version 1.13.
I didn't leave any comments... I'll add some!